www/src/pages/docs/index.js

```diff
@@ -49,6 +49,10 @@ class IndexRoute extends React.Component {
                   around building with Gatsby, like sourcing data, deployment,
                   and more.
                 </li>
+                <li>
+                  <strong>Ecosystem</strong>: Check out libraries for Gatsby
```

Comment:
Not really new here, but is there a reason these aren't linked?

Reply:
I suppose because they're linked in the sidebar? But I'm not sure I see a big issue with doubling up here!

Reply:
It's because very few of these section headers are linked to actual pages (so there's nothing to link to).

Reply:
The point of including this list in the docs introduction landing page is to introduce the section areas more explicitly (people have a better idea of what they're getting).

Reply:
Got it; that makes sense. Thanks!